collector_test.go

```diff
@@ -0,0 +1,54 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package datadog
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+func TestAddViewData(t *testing.T) {
+	exporter := NewExporter(Options{Namespace: "hello", Tags: []string{"test:optionalTag"}})
```

Comment:
The below 4 lines of code can be factored out into a helper function, for example:
```go
func testExporter(opts Options) *Exporter {
    e := NewExporter(opts)
    view.RegisterExporter(exporter)
    view.SetReportingPeriod(time.Millisecond)
}
```
This will remove some duplication and make the tests shorter and easier to follow.